zipkin/pom.xml

```diff
@@ -63,6 +63,7 @@
 
   <build>
     <plugins>
+      <!-- Need to block import of shaded packages as maven bundle plugin analyzes the unshaded jar -->
```

Comment:
maybe remind people the file that blocked imports go is `bnd.bnd`? I think many don't know this